operations/phlare/helm/phlare/values.yaml

```
@@ -710,6 +710,12 @@ phlare:
             regex: Pending|Succeeded|Failed|Completed
             action: drop
 
+  # Allows to add tenant specific overrides to the default limit configuration.
```

If you do this, it should show up in the README.md correctly

```suggestion
  # -- Allows to add tenant specific overrides to the default limit configuration.
```